lib/appsignal/transaction.rb

```diff
@@ -435,7 +435,11 @@ def sanitized_session_data
       session = request.session
       return unless session
 
-      Appsignal::Utils::ParamsSanitizer.sanitize(session.to_hash)
+      options = {}
+      if Appsignal.config[:filter_session_data]
+        options[:filter_parameters] = Appsignal.config[:filter_session_data]
+      end
+      Appsignal::Utils::ParamsSanitizer.sanitize session.to_hash, options
```

Comment:
I'd prefer not to pass a hash here, so I've written a proposal to do that in https://github.com/appsignal/appsignal-ruby/pull/405.